tools/build_variables.bzl

```diff
@@ -496,6 +496,7 @@ libtorch_python_core_sources = [
     "torch/csrc/autograd/python_variable_indexing.cpp",
     "torch/csrc/jit/backends/backend_init.cpp",
     "torch/csrc/jit/backends/backend_resolver.cpp",
+    "torch/csrc/jit/cuda/cuda.cpp",
```

Comment:
I think this should be in `libtorch_core_jit_sources` sources so that it can be used in C++ without a Python dependency.